### PR TITLE
Feat/manager 13152

### PR DESCRIPTION
**Pull request by benjaminlepine**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/pnr-v2
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13152
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by JacquesLarique — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx [hunk 1]
```diff
@@ -212,347 +150,130 @@ const Sidebar = (): JSX.Element => {
...
-          <span
-            className="oui-icon oui-icon-chevron-left"
-            aria-hidden="true"
-          ></span>
-          {t('sidebar_back')}
-        </a>
-      )}
       <div className={style.sidebar_menu}>
         {(servicesCount || betaVersion === 1) && (
-          <ul id="menu">
```
Comment (JacquesLarique) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx:456]: Please keep the id "menu" on the ul, as it is used by the guided tour

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts [hunk 2]
```diff
@@ -17,141 +18,11 @@ const root: Node = {
...
-      id: 'sunrise',
-      translation: 'sidebar_sunrise',
-      routing: {
-        application: 'sunrise',
-      },
-      count: false,
-      features: ['sunrise'],
-      separator: true,
-    },
-    {
```
Comment (JacquesLarique) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts:34]: Not sure where it went but I think this is still needed (maybe in its own file)
For example `sidebar-link-billing` will be used by the guided tour

**Review by JacquesLarique — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/container/common/urls-constants.ts [hunk 3]
```diff
@@ -43,12 +45,14 @@ const urls: URLLinks = {
       WS: `${helpRoot}/es${homeIndex}`,
     },
     status: 'https://www.status-ovhcloud.com/',
+    marketplace: 'https://marketplace.ovhcloud.com/',
```
[the remark this answers is not in the document]
Comment (benjaminlepine) [reply packages/manager/apps/container/src/container/common/urls-constants.ts:48]: i removed it

File: packages/manager/apps/container/src/container/common/urls-constants.ts [hunk 4]
```diff
@@ -43,12 +45,14 @@ const urls: URLLinks = {
...
   },
   US: {
     help: {
       US: 'https://us.ovhcloud.com/support',
     },
     status: 'https://status.us.ovhcloud.com/',
+    marketplace: 'https://marketplace.ovhcloud.com/',
```
[the remark this answers is not in the document]
Comment (benjaminlepine) [reply packages/manager/apps/container/src/container/common/urls-constants.ts:55]: removed

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SidebarLink.tsx [hunk 5]
```diff
@@ -1,28 +1,32 @@
...
   count?: number | boolean;
   node?: Node;
   linkParams?: Record<string, string>;
-  onClick?(): void;
+  handleClick?(): void;
   id?: string;
+  isShortText?: boolean;
 }
 
 const StaticLink: React.FC<ComponentProps<StaticLinkProps>> = ({
```
[the remark this answers is not in the document]
Comment (benjaminlepine) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SidebarLink.tsx:23]: Done

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SubTree.tsx
[the remark this answers is not in the document]
Comment (benjaminlepine) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SubTree.tsx:1]: I think it's a great idea, but it's quite complex to implement. 
We will need to keep it for a separate ticket.

**Review by JacquesLarique — CHANGES_REQUESTED** [review]
The menu is not working as expected: once the mouse leave the sub menu, the sub menu stay display.
Also we need to validate what we do with the user account menu, from the code it seems no longer displayed but we still need links like the identity documents one.

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts [hunk 6]
```diff
@@ -172,6 +172,51 @@ export function findPathToNodeByApp(
...
+    setTimer(
+      setTimeout(() => {
+        func.apply(this, args);
+      }, timeout),
+    );
+  };
+};
+
+export const isMobile = () => {
+  const regex = /Mobi|Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i;
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts:216]: We could use an external lib to be more accurate, but there isn't an optimal way to do that I think

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx [hunk 7]
```diff
@@ -48,26 +37,19 @@ const AssistanceSidebar: React.FC<ComponentProps<Props>> = ({
...
     };
+    const initCarbonCalculator = async () => {
+      const results = await reketInstance.get(
+        `/feature/carbon-calculator/availability`,
+        {
+          requestType: 'aapi',
+        },
+      );
+      setHasCarbonCalculator(results['carbon-calculator']);
+    }
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx:48]: Yes, it's handle globally at the navigation tree level, but we're outside of the tree here.

[the remark this answers is not in the document]
Comment (darsene) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx:48]: In that case, you can group your features in one call.
```
const results: Record<string, boolean> = await reketInstance.get(
        `/feature/livechat,carbon-calculator/availability`,
        {
          requestType: 'aapi',
        },
      );
```

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (oalkabouss) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts:216]: So, well, never mind.

**Review by darsene — APPROVED** [review]

**Review by oalkabouss — APPROVED** [review]

**Pull request merged**